test/local_integration_test.py

```diff
@@ -77,6 +87,30 @@ def check_endpoint_is_working(self, lambda_endpoint: str, url: str):
         url = lambda_endpoint + url
         response = self.requests.get(url)
         response.raise_for_status()
+        # Assert that manifest contains at least one row of metadata.
+        if '/manifest/files' in url:
```

Comment:
This is ugly and only covers `/manifest/files` not `/repository/files/export`. Can you extract this into a separate function: `check_manifest(check_endpoint_is_working(…))`. For that to work, check_endpoint_is_working needs to return `response.content`.

Reply:
Fixed.